README.rst

```diff
@@ -105,6 +102,22 @@ from the github repository or archive, unpack, and run from within the
 toplevel `python-control` directory::
 
   pip install .
+  
+Article and Citation Information
+================================
+
+A `2021 article <https://ieeexplore.ieee.org/abstract/document/9683368>`_ about the library is available on IEEE Explore. If the Python Control Systems Library helped you in your research, please cite::
+
+  @inproceedings{python-control2021,
+    title={The python control systems library (python-control)},
+    author={Fuller, Sawyer and Greiner, Ben and Moore, Jason and Murray, Richard and van Paassen, Ren{\'e} and Yorke, Rory},
+    booktitle={2021 60th IEEE Conference on Decision and Control (CDC)},
```

Comment:
Remove 2021 here, since it appears below.

Reply:
Google Scholar's bibtex downloads are often a little off